### DISCUSSION TITLE
Spare V1 boards for UK

### DISCUSSION THREAD

**Discussion by wyliecoyoteuk** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#362] [General]
Hi, I have 6 V1 PCBs, willing to post to UK for £2 first class post.

**Comment by frogsuk** [comment]
Do you also have a list of components and best UK source for them?

**Reply by wyliecoyoteuk** [reply]
Hi, I used to buy stuff from China, but ebay has a lot of suppliers.
I have bought these items recently:
ebay item numbers below

female header sockets (optional) 264445506779
8ch LLC   (2pcs)                            173404294539
NodeMCU ESP2866                       265114741360


6 pin SM plug connector pair     (from China) 265802295228 couldn't source these in UK from ebay, but available from the Pi Hut
https://thepihut.com/products/6-pin-jst-sm-plug-receptacle-cable-set

**Reply by frogsuk** [reply]
I’ve built my board you kindly supplied. I’m having problems, are you able to help?
Please see my comment **_Please, has anyone got any idea where I’m going wrong?_**

**Reply by stav242** [reply]
Hi. I will take one of you have a spare please. ?

**Reply by wyliecoyoteuk** [reply]
frogsuk, As we haven't heard any more from you, I assume that you now have it working OK?
stav242, yes, I still have some. whereabouts are you?

**Reply by stav242** [reply]
I'm in Dorset. 

stuart tavener gmail com

I've got all the other parts, just need the board now. 

Please ping me an email to sort out payment . Thanks.

**Reply by wyliecoyoteuk** [reply]
Emailed you

**Comment by stav242** [comment]
Hi. Let me know if you can send me one.

Stuart Tavener Gmail com

**Reply by wyliecoyoteuk** [reply]
I have emailed you, check your junk folder

**Comment by seesharpsteve** [comment]
Hi, do you still have any boards available?  I would be very grateful of one if you do.  

Thanks in advance!

**Reply by wyliecoyoteuk** [reply]
Hi I still have a couple, £2 including postage.
email me leojmaxwellsebay at gmail com

**Reply by EgoManiac** [reply]
Hey buddy.. I am just starting on this journey, and in need of a board.. do you happen to have any left?

**Reply by wyliecoyoteuk** [reply]
I have a couple left still.
Email me leojmaxwellsebay at gmail com

**Comment by frogsuk** [comment]
Got all my parts in the post today and soldered it all together. 
Altered model.h to #define PCB_1 and the //#define PCB_2 as picture

Clicked on platformo.ini and have left as is

Then clicked Build File System which all went OK


Then clicked on Upload Filesystem Image and got the folowing errors

Any ideas at all?
I have tried changing the com port in platformio.ini from
;upload_port = COM4
to upload_port =COM8 as that is where the port on my computer 
But still no difference
 “WIFI-remote-for-bestway...” etc so i can see if that works on mine please when you get a moment

**Comment by stav242** [comment]
what errors did you get ?

-cant see anything in your post.
